rholang/main/views.py

```diff
@@ -42,7 +37,7 @@ def home(request):
             run_error = None
 
             if rbl is not None:
-                vm = runner.VM.make(Path(VM_PROGRAM), Popen)
+                vm = runner.VM.make(VM_PROGRAM, VM_LIBRARY, Popen)
```

Comment:
I'm trying to use `Path` objects rather than strings in order to maintain ocap discipline.